howto/tutorial/README.md

```diff
@@ -508,7 +508,6 @@ override func viewDidLoad() {
   [self.view addSubview:self.fab];
 
   self.fab.translatesAutoresizingMaskIntoConstraints = NO;
-  [self.fab.trailingAnchor constraintEqualToAnchor:self.view.trailingAnchor constant:-16.0].active = YES;
   [self.fab.trailingAnchor constraintEqualToAnchor:self.view.trailingAnchor constant: -16.0].active = YES;
```

Comment:
Nit: Delete the second line or else remove the space between `constant: -16.0`

```objectivec
// Before
[self.fab.trailingAnchor constraintEqualToAnchor:self.view.trailingAnchor constant: -16.0].active = YES;
// After
[self.fab.trailingAnchor constraintEqualToAnchor:self.view.trailingAnchor constant:-16.0].active = YES;
```

Reply:
Thanks @romoore , reason I chose the latter and not the former line is because the other constraint has a space between. Is our code style usually without a space? If so, I will go ahead and fix the other constraint spacing in this PR too.

Reply:
Hi @yarneo we usually do *not* put a space before parameters.
Good catch!